packages/back-end/src/api/api.router.ts

```diff
@@ -37,7 +49,13 @@ router.get("/", (req, res) => {
 
 // API endpoints
 router.use("/features", featuresRouter);
+router.use("/experiments", experimentsRouter);
+router.use("/metrics", metricsRouter);
+router.use("/segments", segmentsRouter);
+router.use("/dimensions", dimensionsRouter);
+router.use("/projects", projectsRouter);
 router.use("/sdk-connections", sdkConnectionsRouter);
+router.use("/data-sources", dataSourcesRouter);
```

Comment:
I think we may need CORS enabled on some of these (at least for those that are needed for the Visual Editor), since clients will be making requests from their browser using the Visual Editor.

Reply:
(I can also add it on a follow-up PR, just wanted to run it by you in case you had suggestions.)

Reply:
Hmm.  I kind of wanted to avoid CORS for the REST API to discourage people from using secret keys in a client-side application.  Longer term, it probably makes sense to implement some sort of OAuth flow to authenticate the extension.  Maybe in the short term, we can add an extra undocumented header that's required for CORS headers to be sent back.